apps/web/src/pages/Pools/PoolAdvertisementPage/components/EducationRequirements.tsx

```diff
@@ -105,20 +109,35 @@ const EducationRequirements = () => {
       </span>
       <RequirementCard>
         <Heading level="h4" size="h6" data-h2-margin-top="base(0)">
-          {intl.formatMessage({
-            defaultMessage: "2-year post-secondary",
-            id: "ZIwaDE",
-            description: "Title for the education requirements",
-          })}
+          {isIAP
+            ? intl.formatMessage({
+                defaultMessage: "High school diploma or GED",
+                id: "qF/JU3",
+                description:
+                  "Title for the education requirements (Indigenous apprenticeship program)",
+              })
+            : intl.formatMessage({
+                defaultMessage: "2-year post-secondary",
+                id: "ZIwaDE",
+                description: "Title for the education requirements",
+              })}
         </Heading>
         <Text>
-          {intl.formatMessage({
-            defaultMessage:
-              "Successful completion of two years of post-secondary education in computer science, information technology, information management or another specialty relevant to this position.",
-            id: "RlYe/i",
-            description:
-              "post secondary education experience for pool advertisement",
-          })}
+          {isIAP
+            ? intl.formatMessage({
+                defaultMessage:
+                  "Successful completion of a standard high school diploma or general education development (GED) equivalent.",
```

Comment:
General Education Development is a proper noun and should be title case.

Reply:
Updated as part of [fix typos, translation ids/descriptions](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6668/commits/8b3ae1fd9890817bbe29d235185b3fe9f0318389)